Library/Homebrew/test/cask/utils_spec.rb

```diff
@@ -46,7 +46,7 @@
 
       expect(path).to be_a_file
       expect(link).to be_a_symlink
-      expect(link.realpath).to eq path
+      expect(link.readlink).to eq path
```

Comment:
Any reason to prefer this to
```suggestion
      expect(link.realpath).to eq path.realpath
```
?

Reply:
`readlink` is faster. Though tbh these checks are just testing the `ln_s` 4 lines above so are largely pointless anyway.